lib/vmdb/loggers/container_logger.rb

```diff
@@ -1,5 +1,5 @@
 module Vmdb::Loggers
-  class ContainerLogger < VMDBLogger
+  class ContainerLogger < ManageIQ::Loggers::Base
```

Comment:
If we are switching to manageiq-loggers, then this entire class probably goes away since there is already a container logger there.